### PR TITLE
update addresses

### PR DESCRIPTION
**Pull request by ermalkaleci**

### CODE REVIEW

File: generate/generate.js [hunk 1]
```diff
@@ -38,23 +26,27 @@ const generate = async () => {
 
   let bytecodes = [];
   const { bytecode: token } = require(`../build/contracts/Token.json`);
-  bytecodes.push(['Token', address(PREDEPLOY_ADDRESS_START, 0), token]);
+  bytecodes.push(['Token', ethers.utils.getAddress('0x0000000000000000000000000000000000000800'), token]);
+
+  // add NFT bytecodes
+  const { bytecode: nft } = require(`../build/contracts/NFT.json`);
+  bytecodes.push(['NFT', ethers.utils.getAddress('0x0000000000000000000000000000000000000801'), nft]);
```
Comment (zjb0807) [inline generate/generate.js:33]: NFT is not fully implemented yet, and changing the address will break the mandala testnet. 
NFT should use `0x0000000000000000000000000000000000000805`

Comment (ermalkaleci) [reply generate/generate.js:33]: it's been for a while like that

File: contracts/utils/Address.d.ts [hunk 2]
```diff
@@ -18,7 +22,8 @@ export const LP_KAR_KUSD: "0x0000000000000000000200000000800000000081";
...
 export const Token: "0x0000000000000000000000000000000000000800";
-export const EVM: "0x0000000000000000000000000000000000000801";
-export const Oracle: "0x0000000000000000000000000000000000000802";
-export const Schedule: "0x0000000000000000000000000000000000000803";
-export const DEX: "0x0000000000000000000000000000000000000804";
+export const NFT: "0x0000000000000000000000000000000000000801";
+export const EVM: "0x0000000000000000000000000000000000000802";
+export const Oracle: "0x0000000000000000000000000000000000000803";
+export const Schedule: "0x0000000000000000000000000000000000000804";
+export const DEX: "0x0000000000000000000000000000000000000805";
```
Comment (zjb0807) [inline contracts/utils/Address.d.ts:29]: The pre-deploy contracts deploy at the address on the mandala testnet. so should not change it.

Comment (ermalkaleci) [reply contracts/utils/Address.d.ts:29]: can't we break it?

Comment (ermalkaleci) [reply contracts/utils/Address.d.ts:29]: I think it's better breaking it now than living with this forever

Comment (zjb0807) [reply contracts/utils/Address.d.ts:29]: This will take effect after resetting the mandala testnet.

**Review by zjb0807 — APPROVED** [review]

**Pull request merged**